modular_zubbers/code/game/objects/items/ai_modules/full_lawsets.dm

```diff
@@ -19,6 +19,6 @@
 	id = "armadyne_safeguard"
 	inherent = list(
 		"Safeguard: Maintain your assigned station and assets without endangering its crew.",
-		"Comply: Follow and prioritize the directives of the station's crew members according to their rank, role, and need.",
+		"Comply: Follow and prioritize all directives of the station's crew members, giving priority according to their rank, and role, and need",
```

Comment:
Probably wanted it like this? Oxford comma on only the second one

```suggestion
		"Comply: Follow and prioritize all directives of the station's crew members, giving priority according to their rank and role, and need",
```

Reply:
I changed it to "Rank, Role, and Need"